<h1>Interpretación de Resultados para Ambas Variables</h1>

## Paso IV.a - Variable Continua $Y$

Reutilizamos el modelo Exponencial ajustado en el Paso 3 ($\lambda = 1/\bar y$) y evaluamos su función de distribución acumulada en $y = \bar y + s$:

$$
F_Y(\bar y + s) = 1 - e^{-\lambda(\bar y + s)}
$$

In [2]:
%run scripts/interpretacion_y.py

| Parámetro | Valor |
|---|---|
| $\bar y$ | 6.5781 |
| $s$ | 6.4644 |
| $\bar y + s$ | 13.0425 |
| $\lambda$ | 0.1520 |

<IPython.core.display.Math object>

Porcentaje esperado por debajo de ($\bar y + s$): 86.23%

### Conclusion

Según el modelo Exponencial ajustado a los precios de los juegos de Steam, se espera que aproximadamente el **86.23%** de los juegos tengan un precio por debajo de $\bar y + s \approx \$13.04$ (el precio promedio más una desviación estándar). Es decir, el modelo predice que solo alrededor del 13.77% de los juegos superan ese umbral — un resultado consistente con la forma decreciente observada en el histograma: los precios altos son cada vez menos frecuentes.

# Paso IV.b - Variable Discreta $X$

Se eligió como valor de referencia

$$
k=12
$$

ya que representa una clasificación de edad frecuente en videojuegos y permite conocer qué proporción del catálogo puede ser utilizada por jugadores cuya edad mínima requerida es de **12 años o menos**.

Por definición, la función de distribución acumulada se expresa como:

$$
F_X(x)=P(X\le x)
$$

Por lo tanto,

$$
P(X\le12)=F_X(12)
$$

Utilizando la función de distribución acumulada construida en el Paso II,

$$
F_X(12)=P(X=0)+P(X=3)+P(X=7)+P(X=12)
$$

Sustituyendo las probabilidades puntuales:

$$
F_X(12)=0.977987+0.000406+0.000443+0.002696
$$

Realizando la suma:

$$
F_X(12)=0.981532
$$

Por lo tanto,

$$
\boxed{P(X\le12)=0.981532}
$$

### Conclusión

Esto significa que aproximadamente el **98,15 %** de los videojuegos del dataset requieren una edad mínima de **12 años o menos**.

En consecuencia, puede concluirse que la gran mayoría de los juegos disponibles en Steam presentan una clasificación apta para un público general o con restricciones de edad relativamente bajas, mientras que solo una pequeña proporción exige una edad mínima superior a los 12 años.

# Paso IV.c - Reflexión general

Consideramos que los modelos elegidos representan adecuadamente el comportamiento general de ambas variables. En el caso de la variable discreta (*Required Age*), la función de probabilidad describe correctamente la distribución de las edades mínimas requeridas por los videojuegos. Para la variable continua (*Price*), el modelo exponencial refleja la tendencia general de que la mayoría de los juegos poseen precios bajos y pocos alcanzan valores elevados. Sin embargo, ambos modelos presentan limitaciones, ya que simplifican la realidad y no capturan completamente valores atípicos ni particularidades de la distribución de los datos.